lib/osv/request_helper.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Request helpers"""
+
+import requests
+from requests.adapters import HTTPAdapter, Retry
+
+from .cache import Cache
+
+DEFAULT_BACKOFF_FACTOR = 5
+DEFAULT_RETRY_TOTAL = 7
+DEFAULT_REDIS_TTL_SECONDS = 6 * 60 * 60
+
+
+class RequestException(Exception):
```

Comment:
nit: per https://google.github.io/styleguide/pyguide.html#244-decision, exception class names should end with "Error".